lib/matplotlib/legend.py

```diff
@@ -407,17 +411,6 @@ def __init__(
             List of `.Artist` objects added as legend entries.
 
             .. versionadded:: 3.7
-
-        Notes
-        -----
-        Users can specify any arbitrary location for the legend using the
-        *bbox_to_anchor* keyword argument. *bbox_to_anchor* can be a
-        `.BboxBase` (or derived there from) or a tuple of 2 or 4 floats.
-        See `set_bbox_to_anchor` for more detail.
```

Comment:
And this is described in much more detail in the `bbox_to_anchor` parameter docstring, so I didn't see a need to keep it either.

Reply:
Agreed, and the user comes to the parameter docstring before this.